lib/octokit/client/search.rb

```diff
@@ -20,6 +20,21 @@ def search_code(query, options = {})
         search "search/code", query, options
       end
 
+      # Search commits
+      #
+      # @param query [String] Search terms and qualifiers
+      # @param options [Hash] Sort and pagination options
+      # @option options [String] :sort Sort field
```

Comment:
Maybe `Field to sort by` instead of `Sort field`.

Reply:
I was matching the language of the other option descriptions.

Reply:
Yeah, the rest of the codebase uses `Sort field`. Some endpoints also mention what the available sort options are, but many do not.